### ISSUE TITLE
ERROR: Failed to copy the manifest to the app folder.

### ISSUE THREAD

**Issue by kevincobain2000** [manifoldjs/ManifoldJS#32]
Hi There, am i missing something here from the Example ?

```
manifoldjs http://meteorite.azurewebsites.net -d native  -p ios -b
```

```
05/01/15 @ 3:35PM - pulkit.kathuria@Mac Pro ~/Downloads/temp sudo rm -rf native;sudo manifoldjs http://meteorite.azurewebsites.net -d native  -p ios -b
Validation warning (ios): An app icon of the following sizes is required: 76x76, 120x120, 152x152 and 180x180
Validation warning (ios): An 1024x1024 app icon for the App Store is required
Validation warning (ios): A launch image of the following sizes is required: 750x1334, 1334x750, 1242x2208, 2208x1242, 640x1136, 640x960, 1536x2048, 2048x1536, 768x1024 and 1024x768
ERROR: Failed to copy the manifest to the app folder.
WARNING: The Cordova project could not be created successfully.
WARNING: One or more errors occurred when generating the application.
```

permission (threw in 777)

```
05/01/15 @ 3:39PM - pulkit.kathuria@Mac Pro ~/Downloads ls -la
..
...
drwxrwxrwx   3 pulkit.kathuria  XXX\Domain Users   102 May  1 15:40 temp
```

**Comment by boyofgreen** [comment]
I think the problem here is that you are missing some required images sizes.  I think we have a bug here though because with an iOS project, there should be some default images provided by Cordova, so we should probably let it continue to build.  We also have in our roadmap to add our own default images.
  We'll update issue after I talk with the other devs

**Comment by nbellocam** [comment]
This is now fixed in [v0.1.3](https://github.com/manifoldjs/ManifoldJS/tree/v0.1.3)

**Comment by kevincobain2000** [comment]
@nbellocam awesome Thanks
